36-vending-machine-monitor/firmware/vending-machine/notecard-aux-serial.h

```diff
@@ -0,0 +1,328 @@
+// Copyright 2023 Blues Inc.  All rights reserved.
```

Comment:
I'm concerned that the inheritance and whatnot being used in this file is going to be a pretty big mental hurdle for folks coming from a C programming background. But we can debate that another time. If it's getting the job done, let's leave it as-is for the sake of efficiency in releasing these apps.